roles/deploy/tasks/cleanup.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Remove all dirs, but keep the current version and from the rest the most recent one.
```

Comment:
Most recent two, right? "tail -n +3" displays third line until the end, thus omitting the first two.